cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/template/etl/common/TransformDetails.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.template.etl.common;
+
+import javax.annotation.Nullable;
+
+/**
+ * Class to encapsulate id of the transform and the error dataset used in that transform stage.
+ */
+public class TransformDetails {
```

Comment:
was confused for a little bit because this looks so similar to TransformationDetails.  Lets rename this to TransformInfo or something like that, and TransformationDetails to TransformDetail